codespell_lib/data/dictionary.txt

```diff
@@ -20787,7 +20804,12 @@ plalforms->platforms
 planation->plantation
 plantext->plaintext
 plantiff->plaintiff
-plase->please
+plase->please, place,
+plased->placed, pleased,
```

Comment:
It would be nice to be consistent between please(d) and place(d) first unless one is more likely in each instance.